### PR TITLE
Fix selector for CaaSP VMX flavor

### PR DESCRIPTION
**Pull request by kravciak**
Fixes: https://openqa.suse.de/tests/978484#step/transactional_update/148
Local run:
 - http://dhcp91.suse.cz/tests/5395 - VMX
 - http://dhcp91.suse.cz/tests/5396 - DVD

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -220,13 +220,13 @@ sub is_casp {
     return 1 unless $flavor;
 
     if ($flavor eq 'DVD') {
-        return get_var('FLAVOR') =~ /DVD/;    # Catch DVD and DVD-Staging
+        return get_var('FLAVOR') =~ /DVD/;    # DVD and Staging-?-DVD
     }
     elsif ($flavor eq 'VMX') {
-        return 1;                             # Everything other than DVD is VMX image
+        return get_var('FLAVOR') !~ /DVD/;    # If not DVD it's VMX
```
Comment (dzedro) [inline lib/utils.pm]: How can be this DVD when it is inside `elsif ($flavor eq 'VMX') {` ?

Comment (kravciak) [inline lib/utils.pm]: "!~" negation

Comment (dzedro) [inline lib/utils.pm]: yes, but how can there by anything else than VMX ? Also the change above.
I don't get the "duplicate" condition

Comment (kravciak) [inline lib/utils.pm]: is_casp should tell you if distribution is CaaSP

Optional arguments are:
 - DVD - which returns true if it's ISO image
 - VMX - which returns true if it's virtual image (kvm,vmware,xen,openstack) - everything other than ISO
 - other - which returns true if FLAVOR is matching argument (for is_casp "vmware"...)

**Pull request merged**